### PR TITLE
Implemention of order page api

### PR DESCRIPTION
**Pull request by cs-lee**
1. Add api for order page.

### CODE REVIEW

File: client/order/orderui.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  var iImg = document.createElement('img');
+  var iSend = document.createElement('button');
+  var iText = document.createElement('input');
+
+  /**
+   * API for control order ui
+   */
+  var orderPage = {
+    /**
+     * Initialization of rrder page.
```
Comment (romandev) [inline client/order/orderui.js]: What's rrder?

File: client/order/orderui.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+     * @param String Text to be displayed on order page.
+     */
+    pageLoad(welcomeText) {
```
Comment (romandev) [inline client/order/orderui.js]: Why do we need to make this as public?

File: client/order/orderui.js (hunk 1 above)
Comment (cs-lee) [inline client/order/orderui.js]: I will fix and commit again.

File: client/order/orderui.js (hunk 2 above)
Comment (cs-lee) [inline client/order/orderui.js]: To set user parameters at page loading

**Pull request closed without merging**